docs/setup/configuration.md

```diff
@@ -1,54 +0,0 @@
-## Configuration
-
-### Step 1: Register an OAuth Application in GitLab
```

Comment:
Should this information be relocated?

Reply:
I see, it's already in the README

Reply:
Maybe the PR description can describe what's happening here. Something like "This content has already been moved to README. Just cleaning up files no longer being used."

Reply:
I completely agree that my breadcrumb trail isn't ideal. I hadn't fully appreciated all of the changes needed when I started the PR.